drivers/adc/ad7768-1/ad77681.c

```diff
@@ -272,39 +272,42 @@ uint8_t ad77681_get_frame_byte(struct ad77681_dev *dev)
 int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 				  uint8_t *adc_data)
 {
-	uint8_t crc_calc_buf[4], buf[6], crc, frames_8byte = 0;
+	uint8_t crc_xor_calc_buf[4], buf[6], crc_xor;
 	int32_t ret;
 
-	if (dev->conv_len == AD77681_CONV_24BIT)
-		frames_8byte += 3;
-	else
-		frames_8byte += 2;
-
-	if (dev->crc_sel != AD77681_NO_CRC)
-		frames_8byte++;
-
-	if (dev->status_bit)
-		frames_8byte++;
-
 	buf[0] = AD77681_REG_READ(AD77681_REG_ADC_DATA);
 	buf[1] = 0x00;
 	buf[2] = 0x00;
 	buf[3] = 0x00;
 	buf[4] = 0x00;	/* added 2 more array places for max data length read */
-	buf[5] = 0x00;	/* register address + 3 bytes of data (24bit format) + CRC + Status bit */
+	buf[5] = 0x00;	/* register address + 3 bytes of data (24bit format) + Status bit + CRC */
```

Comment:
nitpick: addition operands are interchangeable so technically no need for this fix.

Reply:
Done with the edits.